compiler/rustc_parse/src/parser/mod.rs

```diff
@@ -97,15 +97,15 @@ macro_rules! maybe_whole {
 #[macro_export]
 macro_rules! maybe_recover_from_interpolated_ty_qpath {
     ($self: expr, $allow_qpath_recovery: expr) => {
-        if $allow_qpath_recovery && $self.look_ahead(1, |t| t == &token::ModSep) {
-            if let token::Interpolated(nt) = &$self.token.kind {
-                if let token::NtTy(ty) = &**nt {
+        if $allow_qpath_recovery
+                    && $self.look_ahead(1, |t| t == &token::ModSep)
+                    && let token::Interpolated(nt) = &$self.token.kind
+                    && let token::NtTy(ty) = &**nt
```

Comment:
The formatting is a bit odd here.

Reply:
Rustfmt stuff and for some reason, #[rustfmt::skip] breaks the build

I can remove this snippet if you want it

Reply:
Rustfmt stuff and for some reason, #[rustfmt::skip] breaks the build

I can remove this snippet if you want it

Reply:
If rustfmt says so...

Reply:
Thank you